main.py

```diff
@@ -79,34 +92,125 @@ def parse_post(url):
         print('This post has no subtittle')
         subtitle = ""
 
-    datetime = post.find_element(By.CLASS_NAME, "post-date").get_attribute('title')
-    like_count = post.find_element(By.CLASS_NAME, "like-count").text
+    datetime = out.strftime("%Y-%m-%d %H:%M:%S")
+    like_count = post.find_element(By.XPATH, '//div[contains(concat(" ",normalize-space(@class)," "),"like-button-container")]//div[@class="label"]').text
```

Comment:
This also needs to be wrapped in try/except, I'll just merge and add it myself
```suggestion
        try:
        like_count = post.find_element(By.XPATH, '//div[contains(concat(" ",normalize-space(@class)," "),"like-button-container")]//div[@class="label"]').text
    except NoSuchElementException:
        print('Failed to find like count')
        like_count = 0
```